From 9c5c18dbf8e1845d349ef7020f8af5bc9b56ed1f Mon Sep 17 00:00:00 2001
From: Pawel Dembicki <paweldembicki@gmail.com>
Date: Fri, 28 Sep 2022 17:14:32 +0200
Subject: [PATCH] arm64: dts: ls1012a-frdm/qds: workaround by updating qspi flash to
 single mode

Update rx and tx bus-width to 1 to use single mode to workaround ubifs
issue found with double mode. (The same method as RDB board)

Signed-off-by: Pawel Dembicki <paweldembicki@gmail.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1012a-frdm.dts | 4 ++--
 arch/arm64/boot/dts/freescale/fsl-ls1012a-qds.dts  | 4 ++--
 2 file changed, 4 insertions(+), 4 deletions(-)

--- a/arch/arm64/boot/dts/freescale/fsl-ls1012a-frdm.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1012a-frdm.dts
@@ -148,8 +148,8 @@
 		spi-max-frequency = <50000000>;
 		m25p,fast-read;
 		reg = <0>;
-		spi-rx-bus-width = <2>;
-		spi-tx-bus-width = <2>;
+		spi-rx-bus-width = <1>;
+		spi-tx-bus-width = <1>;
 	};
 };
 
--- a/arch/arm64/boot/dts/freescale/fsl-ls1012a-qds.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1012a-qds.dts
@@ -186,8 +186,8 @@
 		spi-max-frequency = <50000000>;
 		m25p,fast-read;
 		reg = <0>;
-		spi-rx-bus-width = <2>;
-		spi-tx-bus-width = <2>;
+		spi-rx-bus-width = <1>;
+		spi-tx-bus-width = <1>;
 	};
 };
 
